distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1621,12 +1624,53 @@ private ValidatableResponse assertIngestedDirectoryMonitor(String query, int num
             } catch (InterruptedException e) {
             }
         } while (TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startTime)
-                < TimeUnit.MINUTES.toMillis(1));
+                < TimeUnit.MINUTES.toMillis(3));
```

Comment:
Guessing the type of an extensionless file makes the ingest take a little longer than others. Adjusted the timeout here to reuse the method for the new test with a little bit safer buffer.